qemu/deps/performance/start_testpmd.py

```diff
@@ -3,17 +3,19 @@
 import time
 import locale
 import pexpect
+import subprocess
 
 from six import string_types
 
 
 nic1_driver = sys.argv[1]
 nic2_driver = sys.argv[2]
-nic1 = sys.argv[3]
-nic2 = sys.argv[4]
-cores = int(sys.argv[5])
-queues = int(sys.argv[6])
-running_time = int(sys.argv[7])
+whitelist_option = sys.argv[3]
```

Comment:
Could we append it at the end of the command

Reply:
No, we can not append it at the end of the command. it must be before "--".

Reply:
Seems it does not matter, only if you set it the correct position at the command line Line 36, then it would be fine, where you get the `whitelist_option` does not really matter. But I am also okay with current implementation, although it's not a good one.